frontend/src/Series/Editor/SeriesEditor.js

```diff
@@ -61,6 +61,12 @@ function getColumns(showLanguageProfile) {
       isSortable: true,
       isVisible: true
     },
+    {
+      name: 'sizeOnDisk',
+      label: 'Size',
+      isSortable: true,
+      isVisible: true
```

Comment:
I was thinking this would be optional (personally it's unnecessary information to me here), like it is in the series list. That'd mean this PR requires the UI around makingit optional though. What do you think @Taloth?

Reply:
I would be happy to implement this feature.

Reply:
@markus101 That table has not settings yet though. But I actually looked at the PR earlier and thought the same.

Reply:
Yeah, it's not there, otherwise it'd be trivial,. It should be pretty straight forward following another table as an example.

Reply:
How about implementing an equivalent table options dialog like the one on the history page? I don't think it would be too hard. As for persistence of the column settinngs, I guess it would involve an update of the bank end and db schema. That could perhaps be done in a seperate PR, if it's something wanted?

Reply:
That's what I've been referring to with this comment chain.

It wouldn't require any backend changes, it's purely frontend. I think it makes sense to do it as part of this PR, since it changes the UX.

Reply:
Got it! Hopefully I will be able to implement it this weekend.